specs/release-automation/flow-for-automation.puml

```diff
@@ -0,0 +1,65 @@
+@startuml
+title Release Automation Process
+
+start
+group RA-1: Trigerring the Process
+    :User passes link to excel dictionary on OSF to use for the release;
+    :User passes OSF personal access token;
+    :User triggers release process;
+end group
+
+group RA-2: Creating the Release Files
+    :Retrieve Excel Dictionary;
+    :Parse metadata from files sheet;
+    :Construct file names, ending in ".csv" or ".xlsx";
```

Comment:
Can you expand on this step? I would like to know how your interpolation logic for a template string would work as well as the logic for retrieving the `version` template variable.